tests/test_download.py

```diff
@@ -36,7 +32,8 @@ def test_download(tmp_path):
     # mim install mmcls --yes
     result = runner.invoke(install, [
         'mmcls', '--yes', '-f',
-        'https://github.com/open-mmlab/mmclassification.git'
+        'https://github.com/open-mmlab/mmclassification.git', '-t',
+        str(tmp_path)
```

Comment:
Is the command `mim install mmcls --yes -f https://github.com/open-mmlab/mmclassification.git` still valid after merging the PR?

Reply:
Of course, `-t tmp_path` is to ensure that the packages installed by each test case are isolated and do not affect each other. We can discuss further the changes to the test cases.

Reply:
It is not supported to use `-f` to specify the Github link to install the package, but it can be installed by pip VCS project URL:
`mim install git+https://github.com/open-mmlab/mmclassification.git`